pay_ios/ios/Classes/PaymentHandler.swift

```diff
@@ -169,6 +172,12 @@ class PaymentHandler: NSObject {
     
     return paymentRequest
   }
+
+  func updatePaymentStatus(isSuccess: Bool) {
```

Comment:
I suggest to use a nomenclature that refers to the final step of the payment process only. Using `status` is misleading and gives the impression that this method can be used to update the status at any time.
What about `updatePaymentResult`?

Reply:
makes sense 👍